README.md

```diff
@@ -112,9 +111,26 @@ These are the current pipelines running on the repo:
 
 ## Unit & Accessibility Testing
 
-To run all tests run:
+1. Ensure you have a local postgres instance setup and running with a user `postgres` created.
+1. Install `requirements-dev.txt`
+1. Activate your virtual env: `source .venv/bin/activate`
+1. Run `invoke bootstrap-test-db --database-host=your-db-url` to create a db called "fsd_assess_store_test" in your local postgres instance. This uses the invoke python module to execute tasks defined in `tasks.py`
+1. Run pytest
+- Note you will not see any data inserted from the tests as they run within transactions so are not persisted after the testing session.
 
-    pytest
+## Transactional tests
+These rely on the module `pytest-flask-sqlalchemy` which has good docs on its github page: https://github.com/jeancochrane/pytest-flask-sqlalchemy
+
+The main parts of this framework are invoked in `conftest.py` with the following fixture definitions:
+- `enable_transactional_tests` - This makes all tests use transactions so we don't need to turn it on for each test individually
+- `_db` - this makes the framework use our `db` variable from `db.db`, overriding anywhere it is used during the tests.
```

Comment:
💯 These explanations are great and I will certainly ~~steal~~ add them to the other repos using postgres.

Reply:
Thanks guys!